backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/UserAuthorizationVerifier.java

```diff
@@ -51,24 +54,38 @@ public boolean userHasSiteAdminRole() {
     return id != null && _admins.contains(id.getUsername());
   }
 
-  public boolean userHasPermission(UserPermission permission) {
+  public boolean userHasPermissions(List<UserPermission> permissions) {
     isValidUser();
     Optional<OrganizationRoles> orgRoles = _orgService.getCurrentOrganizationRoles();
     // more troubleshooting help here.
     // Note: if your not reaching this code, then grep for
     // 'AbstractAccessDecisionManager.accessDenied' in
     // spring library AffirmativeBased.java and set a breakpoint there.
     if (orgRoles.isEmpty()) {
-      LOG.warn("Permission request for {} failed. No roles for org defined.", permission);
+      LOG.warn(
+          "Permission request for {} failed. No roles for org defined.", permissions.toString());
       return false;
     }
-    if (!orgRoles.get().getGrantedPermissions().contains(permission)) {
-      LOG.warn("Permissions request for {} failed. Not a granted permission.", permission);
+    // check that all the granted peremssions contain this permission.
+    List<UserPermission> failedChecks =
+        permissions.stream()
+            .filter((permission) -> !orgRoles.get().getGrantedPermissions().contains(permission))
+            .collect(Collectors.toList());
+
+    if (!failedChecks.isEmpty()) {
+      // if failed checks are empty, then user has permission
+      LOG.warn(
+          "Permissions request for {} failed. Not a granted permission.", permissions.toString());
       return false;
     }
+
     return true;
   }
 
+  public boolean userHasPermission(UserPermission permission) {
+    return userHasPermissions(List.of(permission));
+  }
+
   public boolean userIsInSameOrg(UUID userId) {
     isValidUser();
     Optional<OrganizationRoles> currentOrgRoles = _orgService.getCurrentOrganizationRoles();
```

Comment:
This change allows the calling code to check multiple permissions at runtime permissions at the same time. The `PersonService.getPatients()` enforces MULTIPLE permission checks (e.g. user allowed to search and see all users across all facilities).